api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/swagger/api/ApiDocV2Service.java

```diff
@@ -66,7 +66,7 @@ public String transformApiDoc(String serviceId, ApiDocInfo apiDocInfo) {
     private void updateSchemeHostAndLink(Swagger swagger, String serviceId, boolean hidden) {
         GatewayConfigProperties gatewayConfigProperties = gatewayClient.getGatewayConfigProperties();
         String swaggerLink = OpenApiUtil.getOpenApiLink(serviceId, gatewayConfigProperties);
-
+        log.debug("Updating host for service with id: " + serviceId + " to: " + gatewayConfigProperties.getHostname());
```

Comment:
Please use ```{}``` for logging

Reply:
log line comes from merge

Reply:
it's temporarry log message, I added it to help find an error in integration test. And I will remove it after fix.